tictactoeSpecific.md

```diff
@@ -0,0 +1,112 @@
+
+
+# TicTacToe Game Specification
+
+## About the Game:
+
+Tic-Tac-Toe, it's a game of two players, who take turns marking spaces (**X or O**), it can be any symbols)in a 3*3 grid.
```

Comment:
We can use any symbol, but are you providing such an option to choose the symbol of their choice. If yes when does that happen? For everygame or for everyround? Then how do you keep track of someones scores? Also internal logic should be generic to allow such use of different symbols, which will be difficult.